raft.c

```diff
@@ -231,6 +231,28 @@ static void executeLogEntry(RedisRaftCtx *rr, raft_entry_t *entry, raft_index_t
     }
 }
 
+static void raftSendNodeShutdown(raft_node_t *raft_node)
+{
+    if (!raft_node) {
+        return;
+    }
+
+    Node *node = raft_node_get_udata(raft_node);
+    if (!node) {
+        return;
+    }
+
+    if (!ConnIsConnected(node->conn)) {
+        NODE_TRACE(node, "not connected, state=%s", ConnGetStateStr(node->conn));
+        return;
+    }
+
+    if (redisAsyncCommand(ConnGetRedisCtx(node->conn), NULL, NULL,
+                "%s %d", "RAFT.NODESHUTDOWN",
```

Comment:
@tezc sorry for nit picking, but I see the pattern we're using elsewhere is `"RAFT.NODESHUTDOWN %d"`.

Reply:
Right, that's unnecessary, fixed.